### ISSUE TITLE
"More option", "copy", "save" and "Expand" buttons are not accessible using the keyboard: A11y_Visual Studio Code Jupyter Extensions_Home_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-jupyter#17043]
### GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #A11ySev1;#DesktopApp; #Visual Studio Code Jupyter Extensions; #GH-VSCodeJupyterExtensions-Win32-Oct2025; #WCAG2.1.1; #FTP; #Keyboard;

### Environment Details:
Application: Visual Studio Code Jupyter Extensions
Version: 1.105.0-insider (user setup)
OS build :26100.4770
Screen reader
JAWS version: 2025
NVDA: 2024.4.2

Repro Steps
1.Install Jupyter extension app.
2.Launch the Visual studio code.
3.Create new jupyter notbook file and run the code
(# Matplotlib widget
from ipywidgets import *
import numpy as np
import matplotlib.pyplot as plt
 
x = np.linspace(0, 2 * np.pi)
fig = plt.figure()
ax = fig.add_subplot(2, 1, 2)
line, = ax.plot(x, np.sin(x))
 
def update(w=1.1):
    line.set_ydata(np.sin(w * x))
    fig.canvas.draw_idle()
 
interact(update))

4.Verify that "More option", "copy", "save" and "Expand" buttons are not accessible using the keyboard.

### Actual Experience.
"More option", "copy", "save" and "Expand" buttons are not accessible using the keyboard.

### Expected Experience
 "More option", "copy", "save" and "Expand" buttons should be accessible using the keyboard keys.

### User Impact:
Keyboard and screen reader users will not be able to access the controls.

### Attachment

https://github.com/user-attachments/assets/10e3623b-460c-4edf-827c-6c6446e49b7a

**Comment by DonJayamanne** [comment]
Please can you attach screen shots, I'm not sure where or what part of VS Code you are referring to.

**Comment by amunger** [comment]
I'm pretty sure it's custom webview toolbar from the jupyter-renderer at :03 in the video.
Ideally we can just migrate to the built-in renderer and move those into the output menu.

<img width="670" height="346" alt="Image" src="https://github.com/user-attachments/assets/ca9b16cf-b070-4da5-bbb6-3a98648542cd" />

**Comment by amunger** [comment]
copy and save image will now exist in the output menu, contributed from core. 
Open in plot viewer still doesn't have its own command though, so let me know what you want to do with that @DonJayamanne

**Comment by DonJayamanne** [comment]
> Open in plot viewer still doesn't have its own command though, so let 

Can extensions contribute to this.
If we do, then we can have extensions that can create something like plot viewer and we can deprecate the plot viewer in the Jupyter extension.

**Comment by amunger** [comment]
We should be able to contribute to the output menu for specific mime types from an extension to provide the accessible way to do this. 
Those specific buttons come from the renderer extension though, and ideally we can just figure out a way to get rid of those in favor of the output menu without being too disruptive.

**Comment by Mana-swi** [comment]
#a11ydb-keyboard-unreachable;
